easybuild/easyconfigs/t/tornado/tornado-timeouts.patch

```diff
@@ -0,0 +1,15 @@
+diff -Naur tornado.orig/tornado-6.1/tornado/httpclient.py tornado/tornado-6.1/tornado/httpclient.py
```

Comment:
Can you put a comment in the past to indicate what it fixes? What was the error, is there an upstream issue...?